pallets/runtime/tests/src/storage.rs

```diff
@@ -180,6 +180,28 @@ impl User {
     }
 }
 
+impl From<User> for Scope {
+    fn from(user: User) -> Self {
+        Self::Identity(user.did)
+    }
+}
+
+impl From<User> for IdentityId {
+    fn from(user: User) -> Self {
+        user.did
+    }
+}
+
+/// Create a `TrustedIssuer` trusted from a User
+impl From<User> for TrustedIssuer {
+    fn from(user: User) -> Self {
+        Self {
+            issuer: user.did,
+            trusted_for: TrustedFor::Any,
+        }
+    }
+}
```

Comment:
Looking at this again, I'd prefer it if these were methods rather than `From` implementations as that renders what is actually happening more opaque. So for example `alice.scope()` seems more immediately obvious and not much longer either. Also, `alice.into()` where `IdentityId` is expected is just `alice.did` so that can be dropped entirely.